### ISSUE TITLE
Any updates?

### ISSUE THREAD

**Issue by hisacro** [nop90/Guitartuner-3DS#1]
hello any updates on this project or readme is much appreciated

**Comment by johnnytest410** [comment]
i dont think this project is maintained anymore. so there is a small chance of either of that happening
